components/dashboard/src/Login.tsx

```diff
@@ -4,15 +4,32 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
-import { gitpod, gitpodIcon, terminal } from './images';
+import { gitpod, gitpodIcon } from './images';
 import { AuthProviderInfo } from "@gitpod/gitpod-protocol";
 import { useContext, useEffect, useState } from "react";
 import { UserContext } from "./user-context";
 import { getGitpodService, gitpodHostUrl, reconnectGitpodService } from "./service/service";
 import { iconForAuthProvider, simplifyProviderName } from "./provider-utils";
+import automate from "./images/welcome/automate.svg";
+import code from "./images/welcome/code.svg";
+import collaborate from "./images/welcome/collaborate.svg";
+import customize from "./images/welcome/customize.svg";
+import fresh from "./images/welcome/fresh.svg";
+import prebuild from "./images/welcome/prebuild.svg";
+
+
+
```

Comment:
**thought**: I will keep a reference of these three blank lines to support our linting approach in the future. 😇

Reply:
🤓